modal/_functions.py

```diff
@@ -1676,7 +1677,9 @@ async def _spawn_map(self, input_queue: _SynchronizedQueue) -> "_FunctionCall[Re
             input_queue,
             self.client,
         )
-        fc: _FunctionCall[ReturnType] = _FunctionCall._new_hydrated(function_call_id, self.client, None)
+        fc: _FunctionCall[ReturnType] = _FunctionCall._new_hydrated(
```

Comment:
quick quality of life improvement that would have made debugging this a bit easier (no repr = defaults to saying it is .from_id() which isn't technically true)